data/test-cases/plugins/ms-teams-sync/attachments-sync/Multiple_attachments.md

```diff
@@ -0,0 +1,49 @@
+---
+# (Required) Ensure all values are filled up
+name: "Send multiple attachments"
+status: Active
+priority: Normal
+folder: attachments-sync
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Upload multiple attachments in linked channel or DM/GM in MM.
```

Comment:
Make these steps consistent for every test case, please refer the steps in the above test case